### PR TITLE
feat: ws2 3034 create node sync graphql test

### PR DESCRIPTION
**Pull request by samika98**
Description : 
Introduces an E2E test ensuring that documents created using the ComposeDB API on Node 1 are properly synchronized and received by Node 2 through the rust-ceramic. This test is for maintaining the reliability of our model syncing capabilities in a distributed setup.

### CODE REVIEW

File: suite/update.sh [hunk 1]
```diff
@@ -26,7 +29,10 @@ do
     scope=$(dirname $dep)
     case $scope in
         "@ceramicnetwork")
-            filtered_deps="$filtered_deps $dep@$target"
+            target_version=$(npm view $dep@$target version)
+            if [[ -n "$target_version" ]] && semver -r ">$current_version" $target_version &> /dev/null; then
```
Comment (nathanielc) [inline suite/update.sh:33]: Will semver be installed? If not maybe we can install it as a dev dependency in packages.json?

**Review by nathanielc — APPROVED** [review]
Looks great, one question below on how this will work in CI.

File: suite/update.sh (hunk 1 above)
Comment (samika98) [reply suite/update.sh:33]: yeah, added this as devDependecy just to be sure

**Review by JulissaDantes — APPROVED** [review]
LGTM!

**Pull request merged**